src/sources/sources.js

```diff
@@ -0,0 +1,50 @@
+import './sources.scss';
+
+export default class Sources {
+  async getSourcesAsync(url) {
+    let response = await fetch(url);
+    let data = await response.json();
+    return data;
+  }
+
+  init(handler) {
+    const urlSources = 'https://newsapi.org/v2/sources?apiKey=83c8aebd6ec444179f992b3fc49b9b3f';
```

Comment:
I see apiKey in the code for not for the first time. Worth to put in the configuration

Reply:
created constants.js